dev/Scroller/TestUI/ScrollerPage.xaml.cs

```diff
@@ -92,10 +92,11 @@ public ScrollerPage()
 
                 txtMouseWheelInertiaDecayRate.Text = ScrollerTestHooks.MouseWheelInertiaDecayRate.ToString();
 
-                chkIsInteractionTrackerMouseWheelZoomingEnabled.IsChecked = (bool)ScrollerTestHooks.IsInteractionTrackerMouseWheelZoomingEnabled;
+                chkIsInteractionTrackerPointerWheelRedirectionEnabled.IsChecked = (bool)ScrollerTestHooks.IsInteractionTrackerPointerWheelRedirectionEnabled;
```

Comment:
type cast not required ?

Reply:
It is, because IsChecked is a nullable bool.

Reply:
really ? Isn't the rhs already a bool ?

Reply:
You were right - it's the other way around that would need the cast.